src/fluree/db/query/analytical_wikidata.cljc

```diff
@@ -34,6 +34,15 @@
                  res)
                res)))
 
+(defn get-all-wd-optional-clauses
+  [coll]
+  (reduce (fn [res clause]
+            (if-let [optional (-> clause :optional)]
```

Comment:
This is very minor, but the thread macro (`->`) is typically only used when there is more than one function to apply, but here, there's just one. 

This line could be:
```suggestion
            (if-let [optional (:optional clause)]
```

Or you could just destructure in the reducing function definition:
```clojure
(reduce (fn [res {:keys [optional] :as clause}]
  (if optional ...
```

Reply:
opted for the destructuring

Also, created a ticket for wikidata-type unit tests because we should not be dependent on the availability of wikidata site.